generator/src/lib.rs

```diff
@@ -1123,6 +1146,40 @@ pub fn generate_extension<'a>(
     };
     Some(q)
 }
+pub fn generate_define(define: &vkxml::Define) -> Tokens {
+    // TODO: Would rather leverage generate_constant
```

Comment:
you can also remove the todos

Reply:
Yeah, I don't think we'll refactor the code surrounding this. Tried working on a tokenstream too, while that might be advantageous if operating on more complex expressions in the future, it's too much overhead right now.